sdk/eventhub/event-processor-host/package.json

```diff
@@ -58,7 +58,7 @@
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "dependencies": {
-    "@azure/event-hubs": "^2.1.1",
+    "@azure/event-hubs": "^5.0.0-preview.6",
```

Comment:
`event-processor-host` library is Track 1.
The Track 2 version of it is part of the event-hubs library itself.
So, no change to this package needed.

Reply:
Fixed.